.gitignore

```diff
@@ -15,9 +15,16 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# Release artifacts
+adbc-*.tar.gz
```

Comment:
How about using `apache-arrow-adbc-*.tar.gz` because our all releases use `apache-arrow-` prefix?
See also: https://dist.apache.org/repos/dist/release/arrow/

Reply:
Hmm, should we just update everything then (I think tag names are the other place)?

Reply:
Went ahead and updated the tag names too (I'll give this a test run in a moment)

Reply:
Thanks!